src/config.js

```diff
@@ -1,17 +1,36 @@
+let ZERO_ADDRESS = "0x0000000000000000000000000000000000000000";
```

Comment:
i guess this constant could be under config object? it can be utilised in index.js if required in future?